enterprise/cli/provisionerdaemons.go

```diff
@@ -137,9 +137,7 @@ func (r *RootCmd) provisionerDaemonStart() *clibase.Cmd {
 				cliui.Errorf(inv.Stderr, "Unexpected error, shutting down server: %s\n", exitErr)
 			}
 
-			shutdown, shutdownCancel := context.WithTimeout(ctx, time.Minute)
-			defer shutdownCancel()
-			err = srv.Shutdown(shutdown)
+			err = srv.Shutdown(ctx)
```

Comment:
It will do what we describe earlier in this file:

`"Interrupt caught, gracefully exiting. Use ctrl+\\ to force quit",` that is, we'll wait indefinitely until we get a more aggressive signal like SIGQUIT (`ctrl + \`) or SIGKILL.